src/components/trade/PriceSuggestions/PriceSuggestionItem.tsx

```diff
@@ -28,16 +28,26 @@ interface FormattedPrices {
 
 const LOW_PRICE_FLOOR = new BigNumber('0.0001')
 
-export function formatPriceToPrecision(price: BigNumber, useThreshold = false): string {
+interface FormatPriceOptions {
+  useThreshold?: boolean | undefined
+  decimals?: number | undefined
+}
+
+export function formatPriceWithFloor(
+  price: BigNumber,
+  { useThreshold = false, decimals = PRICE_ESTIMATION_PRECISION }: FormatPriceOptions,
```

Comment:
what is more common, to use the threshold or not?
I would think that is more common to default to true, right? (not sure)

Reply:
removed this as in the end it was only used here